src/hardware/gus.cpp

```diff
@@ -682,6 +685,8 @@ Gus::Gus(const io_port_t port_pref, const uint8_t dma_pref, const uint8_t irq_pr
 	SetupEnvironment(port_pref, ultradir);
 
 	LOG_MSG("GUS: Running on port %xh, IRQ %d, and DMA %d", port_pref, irq1, dma1);
+
+	MIXER_UnlockMixerThread();
```

Comment:
Ok, so you're putting the lock/unlock at the first and last lines. Probably reducing the scope wouldn't improve anything, ok.

Reply:
As discussed above, these locks are fine.